### ISSUE TITLE
Where is the rgb2y function?

### ISSUE THREAD

**Issue by Yuanlin-Zhao** [megvii-research/WACV2024-SAFA#5]
train.py shows that this function is exploding; please tell me where to find it. No such file or directory: 'data/adobe240fps_folder_train.txt' Where can I get this file？

**Comment by hzwer** [comment]
The function is Added know.
https://github.com/CS-GangXu/TMNet/blob/main/data/README.md#adobe240fps-dataset

**Comment by Yuanlin-Zhao** [comment]
Thank you very much for your answer

**Comment by Yuanlin-Zhao** [comment]
What is the resolution of the images or videos in the training set and validation set respectively?

**Comment by hzwer** [comment]
adobe240fps
training: 128x patch, 0.25x->1x
validation: 720p, 0.25x->1x
